components/BaseModal/BaseModal.tsx

```diff
@@ -39,6 +39,7 @@ const BaseModal: FC<BaseModalProps> = ({
 			>
 				<StyledModal>
 					<StyledModalHeader>
+						<StyledTitle>{title ?? ''}</StyledTitle>
```

Comment:
`{!!title && <StyledTitle>}` is probably cleaner? Or do you need the Title tag to be here all the time even with an empty string?

Reply:
It's becase the `justify-content: space-between` of the parent element. Now the modal's X icon is in the same place as the title. Ref: https://github.com/AelinXYZ/aelin-ui/pull/228/files#diff-1ceb43e0f8d5a8f97e28a984d953523443e047c11b8fb6fdcbac09b64934a6b2R74

Reply:
ok!